dimagi/pages/templates/sections/sectors/disease-treatment/news.html

```diff
@@ -0,0 +1,146 @@
+{% load dimagidotcom %}
+{% load i18n %}
+
+<section class="pad-bottom-medium"
+         id="news">
+  <div class="content pad-top-medium align-center">
+    <h2>
+      {% blocktrans %}
+        The Latest on Mobile Data Collection in <br class="hide-until-medium"> Disease Treatment Programs
+      {% endblocktrans %}
+    </h2>
+    <p>
+      {% blocktrans %}
+        Updates and advice from our world-class partners and Global Services team
+      {% endblocktrans %}
+    </p>
+  </div>{# .content #}
+  <div class="content pad-top-line news-images">
+    <div class="grid grid-3-at-large grid-1-at-medium grid-row-equal-heights ">
+      <div class="column">
+        <div class="pad-bottom-base white-bg">
+          <div class="image blog1" data-lazybg="true"></div>
+          <div class="pad-line">
+            <h5>
+              <strong>
+                {% blocktrans %}
+                  Leveraging Mobile Data Collection in the Fight Against HIV/AIDS
+                {% endblocktrans %}
+              </strong>
+            </h5>
+            <p class="pad-bottom-base-at-large pad-bottom-base-at-extra-large">
+              {% blocktrans %}
+                A Peace Corps volunteer explains how CommCare aided in a patient data collection
+                program in Cameroon focused on HIV treatment around World AIDS Day.
+              {% endblocktrans %}
+            </p>
+            <p class="pad-top-small-at-large pad-top-small-at-extra-large">
+              <a href="https://www.dimagi.com/blog/why-peace-corps-cameroon-is-leveraging-mobile-data-collection-in-the-fight-against-hiv-aids/"
+                 class="link-gray read-more"
+                 target="_blank">
+                {% blocktrans %}
+                  Read More &gt;
+                {% endblocktrans %}
+              </a>
+            </p>
+          </div> {# .pad-line #}
+        </div> {# .white-bg #}
+      </div> {# .column #}
+      <div class="column">
+        <div class="pad-bottom-base white-bg">
+          <div class="image blog2" data-lazybg="true"></div>
+          <div class="pad-line">
+            <h5>
+              <strong>
+                {% blocktrans %}
+                  USAID Awards Dimagi & Partners a Grand Challenge to Combat Zika and Future Outbreaks
+                {% endblocktrans %}
+              </strong>
+            </h5>
+            <p>
+              {% blocktrans %}
+                USAID announced that Dimagi and partners have been awarded a Combating Zika and Future
+                Threats Grand award — joining a select group of innovators focused on curbing infectious
+                disease worldwide.
+              {% endblocktrans %}
+            </p>
+            <p>
+              <a href="https://www.dimagi.com/blog/usaid-awards-dimagi-partners-a-grand-challenge-to-combat-zika-and-future-outbreaks/"
```

Comment:
use `{% url 'blog_post' 'usaid-awards-dimagi-partners-a-grand-challenge-to-combat-zika-and-future-outbreaks' %}`